pylintrc

```diff
@@ -47,6 +47,8 @@ enable=
     unused-import,
     unused-variable,
     useless-object-inheritance,
+    del-method-used,
+    exception-in-del,
```

Comment:
Please keep these sorted alphabetically. Not having them sorted makes it much too easy to overlook something. (e.g. how we had the assert checker accidentally disabled for nearly a year)

Reply:
Done